Havit/Havit/Screens/CategoryContents/View/Components/Cells/CategoryFilterCollectionViewCell.swift

```diff
@@ -45,6 +47,18 @@ final class CategoryFilterCollectionViewCell: BaseCollectionViewCell {
         super.init(frame: .zero)
     }
     
+    override var isSelected: Bool {
+        didSet {
+            if isSelected {
+                contentView.backgroundColor = .primaryBlack
+                filterNameLabel.textColor = .white
+            } else {
+                contentView.backgroundColor = .whiteGray
+                filterNameLabel.textColor = .gray003
+            }
```

Comment:
p5;

```suggestion
     contentView.backgroundColor = isSelected ? .primaryBlack : .whiteGray
     filterNameLabel.textColor = isSelected ? .white : .gray003
```

이렇게 삼항연산자 사용해서 줄여쓸 수도 있지 않을까유

Reply:
d4e019a589965cd0e190d214